mu4/scenes/inspector/models/text/textsettingsmodel.h

```diff
@@ -74,7 +73,7 @@ public slots:
 
 private:
     void updateFramePropertiesAvailability();
-    void updateInsertSpecialCharAvailability(const Ms::ScoreState &state);
+    void updateInsertSpecialCharAvailability(const Ms::ScoreState& state);
```

Comment:
Can we get rid of `Ms::ScoreState`? This dependence on `mscore`, it is necessary that it was not.
It seems like it uses a check for several states, I think you can replace them with methods like `isSomeThing` in the adapter.

Reply:
Generally, I can hide it in the adapter